packages/app/src/modules/theme.js

```diff
@@ -25,12 +25,29 @@ export const theme = {
                 transition: null,
             },
         },
-        MuiDialogContent: {
-            root: {
-                ...dhis2theme.overrides.MuiDialogContent.root,
-                overflow: 'hidden',
-                overflowY: 'hidden',
+        MuiSelect: {
+            select: {
+                '&:focus': {
+                    background: '$labelcolor',
+                },
+            },
+        },
+        MuiDialog: {
+            paperWidthSm: {
+                minWidth: 400,
+                maxWidth: 400,
             },
+            paperWidthMd: {
+                minWidth: 600,
+                maxWidth: 600,
+            },
+            paperWidthLg: {
+                minWidth: 800,
+                maxWidth: 800,
```

Comment:
We should test all of these in various browsers to ensure they're respecting minWidth at various screen sizes.  I think FF might do some weird things, but I might be mis-remembering.